po/es.po

```diff
@@ -21352,6 +21380,7 @@ msgid "`Cargo.toml`:"
 msgstr "`Cargo.toml`:"
 
 #: src/exercises/concurrency/chat-app.md:19
+#, fuzzy
```

Comment:
Just a note to @dyoo: We should make a new mdbook-i18n-helpers release and start adding `mdbook-xgettext: skip` comments to the course.